zhaquirks/smartwings/wm25lz.py

```diff
@@ -0,0 +1,105 @@
+"""Device handler for Smartwings blinds."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+from zigpy.zcl.clusters.closures import WindowCovering
+from zigpy.zcl.clusters.general import (
+    Basic,
+    Groups,
+    Identify,
+    Ota,
+    PowerConfiguration,
+    Scenes,
+)
+
+from zhaquirks import DoublingPowerConfigurationCluster
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+
+class InvertedWindowCoveringCluster(CustomCluster, WindowCovering):
+    """WindowCovering cluster implementation.
+
+    This implementation inverts the reported covering percent for non standard
+    devices that don't follow the reporting spec.
+    """
+
+    cluster_id = WindowCovering.cluster_id
+    CMD_GO_UP = 0x0001  # reported as down_close in ZHA commands
+    CMD_GO_DOWN = 0x0000  # reported as up_open in ZHA commands
+
+    async def command(
+        self,
+        command_id,
+        *args,
+        manufacturer=None,
+        expect_reply=True,
+        tsn=None,
+        **kwargs
+    ):
+        """Override default commands for up and down. they are backwards."""
+
+        if command_id == self.CMD_GO_UP:
+            return await super().command(self.CMD_GO_DOWN, *args)
+        if command_id == self.CMD_GO_DOWN:
+            return await super().command(self.CMD_GO_UP, *args)
```

Comment:
I guess it already works like that, but do we want to keep the rest of the parameters?
For example:
```python
return await super().command(
    self.CMD_GO_UP,
    *args,
    manufacturer=manufacturer,
    expect_reply=expect_reply,
    tsn=tsn,
    **kwargs,
)
```

or to avoid copying that three times, maybe something like this could be done to just override the ``command_id``:
```python
if command_id == self.CMD_GO_UP:
    command_id = self.CMD_GO_DOWN
elif command_id == self.CMD_GO_DOWN:
    command_id = self.CMD_GO_UP

return await super().command(
    command_id,
    *args,
    manufacturer=manufacturer,
    expect_reply=expect_reply,
    tsn=tsn,
    **kwargs,
)
```
I think that might be a bit better/cleaner